CHANGELOG.md

```diff
@@ -4,6 +4,49 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.4.0
+
+_August 20, 2024_
+
+FEATURES:
+
+- Support IPv6. [2190](https://github.com/nginxinc/nginx-gateway-fabric/pull/2190)
+- Add support for TLS Passthrough using TLSRoutes. [2356](https://github.com/nginxinc/nginx-gateway-fabric/pull/2356)
+- Support cross-namespace routing with TLSRoutes. [2379](https://github.com/nginxinc/nginx-gateway-fabric/pull/2379)
+- Added server_zone metrics for NGINX Plus users. [2360](https://github.com/nginxinc/nginx-gateway-fabric/pull/2360)
+
+BUG FIXES:
+
+- Fixed issue where NGF Pod cannot recover if NGINX master process fails without cleaning up. [2131](https://github.com/nginxinc/nginx-gateway-fabric/pull/2131)
+- Leader election couldn't be disabled. [2307](https://github.com/nginxinc/nginx-gateway-fabric/pull/2307)
+- Disallow routes from attaching to listeners if not present in allowed routes. [2314](https://github.com/nginxinc/nginx-gateway-fabric/pull/2314)
+- Honor ReferenceGrants that allow GRPCRoutes to reference Services in different namespaces. [2337](https://github.com/nginxinc/nginx-gateway-fabric/pull/2337)
+- Fixed an issue that prevented ClientSettingsPolicies and ObservabilityPolicies from working when attached to an HTTPRoute where matching conditions were defined. [2318](https://github.com/nginxinc/nginx-gateway-fabric/pull/2318)
+
+DOCUMENTATION:
```

Comment:
Should we add:

- fix: update kubectl exec syntax to remove deprecation warning by @aknot242 in https://github.com/nginxinc/nginx-gateway-fabric/pull/2218 
- docs: adds info on setting up host network access by @fardarter in https://github.com/nginxinc/nginx-gateway-fabric/pull/2263


since they are external contributors (one is from f5, but still worth mentioning I think)